### DISCUSSION TITLE
RPI Browser setting

### DISCUSSION THREAD

**Discussion by ceejayemm** [rern/rAudio#158] [Fixed Issues] [answered]
After applying 202101311 the RPi Browser option (RPi 3B+) option in Features is turned off and it won't let me turn it back on again. If I turn it on and go to another screen and then back to this setting it is again turned off.

Chris

**Comment by ceejayemm** [comment]
@rern 
This is still an issue after upgrading to the latest image, updated to 202101311 and with 'pacman -Syu' used to upgrade to kernel 5.10.11-4-ARCH armv7l. This installation can no longer be controlled via the official 7in touchscreen without this option available.

Chris

**Reply by rern** [reply]
Post a result of:
`/opt/vc/bin/tvservice -l`

**Reply by ceejayemm** [reply]
@rern 
As requested:

RAConservatory:~  #  /opt/vc/bin/tvservice -l
2 attached device(s), display ID's are :
Display Number 0, type Main LCD
Display Number 3, type Composite
RAConservatory:~  #

Chris

**Reply by ceejayemm** [reply]
@rern 
Update 20210204 doesn't seem to fix this problem I am afraid due to same error reported in #160 

Chris

**Reply by rern** [reply]
Then:
```sh
pacman -Q chromium
systemctl restart localbrowser
systemctl status localbrowser
```

**Reply by ceejayemm** [reply]
@rern 
As requested:

RAConservatory:~  #  pacman -Q chromium
chromium 88.0.4324.146-2
RAConservatory:~  #  systemctl restart localbrowser
RAConservatory:~  #  systemctl status localbrowser
* localbrowser.service - Local Browser - Chromium
     Loaded: loaded (/etc/systemd/system/localbrowser.service; enabled; vendor preset: disabled)
     Active: active (running) since Thu 2021-02-04 11:50:28 UTC; 3s ago
   Main PID: 5088 (sudo)
      Tasks: 73 (limit: 2138)
     CGroup: /system.slice/localbrowser.service
             |-5088 /usr/bin/sudo /usr/bin/startx
             |-5096 /bin/sh /usr/bin/startx
             |-5111 xinit /etc/X11/xinit/xinitrc -- /etc/X11/xinit/xserverrc :0 -auth /tmp/serverauth.PlM1lGRcro
             |-5112 /usr/lib/Xorg -nolisten tcp :0 -auth /tmp/serverauth.PlM1lGRcro
             |-5127 /bin/bash /etc/X11/xinit/xinitrc
             |-5134 matchbox-window-manager -use_cursor no
             |-5135 /usr/lib/chromium/chromium --app=http://localhost --kiosk --incognito --no-sandbox --force-device-scale-factor=1
             |-5137 /usr/lib/chromium/chromium --type=zygote --no-zygote-sandbox --no-sandbox --force-device-scale-factor=1
             |-5138 /usr/lib/chromium/chromium --type=zygote --no-sandbox --force-device-scale-factor=1
             |-5172 /usr/lib/chromium/chromium --type=gpu-process --field-trial-handle=7829459218040257391,13303392933052272894,131>
             |-5174 /usr/lib/chromium/chromium --type=utility --utility-sub-type=network.mojom.NetworkService --field-trial-handle=>
             `-5180 /usr/lib/chromium/chromium --type=utility --utility-sub-type=storage.mojom.StorageService --field-trial-handle=>

Feb 04 11:50:28 RAConservatory sudo[5112]:         (WW) warning, (EE) error, (NI) not implemented, (??) unknown.
Feb 04 11:50:28 RAConservatory sudo[5112]: (==) Log file: "/var/log/Xorg.0.log", Time: Thu Feb  4 11:50:28 2021
Feb 04 11:50:28 RAConservatory sudo[5112]: (==) Using config directory: "/etc/X11/xorg.conf.d"
Feb 04 11:50:28 RAConservatory sudo[5112]: (==) Using system config directory "/usr/share/X11/xorg.conf.d"
Feb 04 11:50:28 RAConservatory sudo[5113]: modprobe: FATAL: Module g2d_23 not found in directory /lib/modules/5.10.11-4-ARCH

Feb 04 11:50:29 RAConservatory sudo[5134]: matchbox-window-manager: Failed to connect to session manager
Feb 04 11:50:32 RAConservatory sudo[5135]: [5135:5175:0204/115032.015767:ERROR:bus.cc(393)] Failed to connect to the bus: Could not>
Feb 04 11:50:32 RAConservatory sudo[5135]: [5135:5175:0204/115032.015923:ERROR:bus.cc(393)] Failed to connect to the bus: Could not>
Feb 04 11:50:32 RAConservatory sudo[5135]: [5135:5175:0204/115032.253175:ERROR:bus.cc(393)] Failed to connect to the bus: Could not>
Feb 04 11:50:32 RAConservatory sudo[5135]: [5135:5175:0204/115032.253302:ERROR:bus.cc(393)] Failed to connect to the bus: Could not>
Feb 04 11:50:32 RAConservatory sudo[5174]: [5174:5200:0204/115032.643502:ERROR:socket_posix.cc(94)] CreatePlatformSocket() failed: >

Please note I am also now using the DIY ROS installation on this machine and also have issue #160 which is causing problems in the same Functions area.

Chris

**Reply by ceejayemm** [reply]
Cut and paste chopped off some text - see attached file for full detail (but its slightly different due to second run of systemctl status localbrowser

Chris
[chrome1.txt](https://github.com/rern/rAudio-1/files/5925549/chrome1.txt)

**Comment by ceejayemm** [comment, marked as answer]
@rern 
This problem has been fixed with one of the recent updates (not sure which of 20210204/05/051) but I still can't configure the browser settings due to the problem reported in #160

Chris
